Formula/b/backtrace_crashpad.rb

```diff
@@ -0,0 +1,77 @@
+class BacktraceCrashpad < Formula
+  desc "Backtrace version of Crashpad with file attachment support and other improvements"
+  homepage "https://github.com/backtrace-labs/crashpad"
+  url "https://github.com/backtrace-labs/crashpad.git",
+      tag:      "v0.1.0",
+      revision: "031c47b4b3349d784d330f670360bb8af0266da3"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  # TODO: Creating iOS variants for the client at some point
+  # TODO: Add minidump_stackwalk once Backtrace supports it
+  def install
+    system "cmake", "-S", ".", "-B", "out", *std_cmake_args
+    system "cmake", "--build", "out"
```

Comment:
Is there no CMake install?

Reply:
Nope, but maybe @rick-bt @gm4sl would be interesting in addressing this upstream? (we would be happy to send a PR)